packages/mongo/mongo_driver.js

```diff
@@ -131,14 +131,17 @@ MongoConnection = function (url, options) {
   self._onFailoverHook = new Hook;
 
   var mongoOptions = _.extend({
-      db: { safe: true },
-      // Set reconnectTries to 0 which means keep trying to reconnect forever,
-      // rather than the default of losing the connection permanently after 30
-      // retries (separated by 1000ms).
-      server: { reconnectTries: 0 },
-      replSet: {}
+    db: { safe: true },
+    // http://mongodb.github.io/node-mongodb-native/2.2/api/Server.html
+    server: {
+      // Reconnect on error.
+      autoReconnect: true,
```

Comment:
It ultimately gets passed in here: https://github.com/christkv/mongodb-core/blob/2.0/lib/topologies/server.js#L35